src/js/main.js

```diff
@@ -77,11 +92,27 @@ class Timelines extends React.Component {
 		{this.homeTimelineResult()};
 		{this.userTimelineResult()};
 		return e('div', {className: 'Timelines'},[
-			e(TimelineComp, {key: 'homeTimelineComp', timelineType: 'Home', buttonFunc: () => getHomeTimeline(this.homeTimelineCallback), resultClass: homeTimelineResultClass, resultOutput:homeTimelineResultOutput}, null),
-			e(TimelineComp, {key: 'userTimelineComp',timelineType: 'User', buttonFunc: () => getUserTimeline(this.userTimelineCallback), resultClass: userTimelineResultClass, resultOutput:userTimelineResultOutput}, null)]);			
+			e(TimelineComp, {onKeyPressButton: this.handleKeyPress, 
+							 onChangeButton: this.handleChange, 
+							 filter: this.state.value, 
+							 key: 'homeTimelineComp', 
+							 timelineType: 'Home', 
+							 buttonFunc: () => getHomeTimeline(this.homeTimelineCallback), 
+							 filterButtonFunc: () => getFilterTimeline(this.homeTimelineCallback, this.state.value), 
+							 resultClass: homeTimelineResultClass, 
```

Comment:
what do resultClass / resultOutput represent?

Reply:
result output represent the timeline object, the large timeline chart
result class represents success, failure, pending, empty timeline